<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignDlg</class>
 <widget class="QDialog" name="SignDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <widget class="QComboBox" name="mKeyTypeCombo"/>
   </item>
   <item row="4" column="5">
    <widget class="QLineEdit" name="mObjectText"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Mechanism</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QComboBox" name="mMechCombo"/>
   </item>
   <item row="6" column="3" colspan="3">
    <widget class="QLineEdit" name="mParamText"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Key Type</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Slot List</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="6">
    <widget class="QTextBrowser" name="mOutputText"/>
   </item>
   <item row="10" column="5">
    <widget class="QPushButton" name="mCloseBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="7" column="2" colspan="4">
    <widget class="QLineEdit" name="mInputText"/>
   </item>
   <item row="1" column="0" colspan="6">
    <widget class="QComboBox" name="mSlotsCombo"/>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Object</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Slot ID</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mSlotIDText"/>
     </item>
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Session</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mSessionText"/>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="mLoginText"/>
     </item>
    </layout>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QComboBox" name="mLabelCombo"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Label</string>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Param</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QComboBox" name="mInputCombo"/>
   </item>
   <item row="9" column="0" colspan="6">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="mInitBtn">
       <property name="text">
        <string>Init</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mUpdateBtn">
       <property name="text">
        <string>Update</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mFinalBtn">
       <property name="text">
        <string>Final</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="mSignBtn">
       <property name="text">
        <string>Signature</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QLabel" name="mStatusLabel">
     <property name="text">
      <string>Status</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
